server/src/main/java/org/opensearch/extensions/rest/RestSendToExtensionAction.java

```diff
@@ -160,17 +161,26 @@ public String executor() {
                 return ThreadPool.Names.GENERIC;
             }
         };
+
         try {
             // Will be replaced with ExtensionTokenProcessor and PrincipalIdentifierToken classes from feature/identity
             final String extensionTokenProcessor = "placeholder_token_processor";
             final String requestIssuerIdentity = "placeholder_request_issuer_identity";
 
+            final Set<String> allowList = Set.of("Content-Type");
+            final Set<String> denyList = Set.of("Authorization", "Proxy-Authorization");
```

Comment:
Move all this declaration/initialization this to the class level and make them static.

Reply:
> Move all this declaration/initialization this to the class level and make them static.

Should it be final?